ebusd-2.1.x/de/vaillant/_templates.csv

```diff
@@ -32,17 +32,17 @@ integral,SIN,,°min,
 hcmode,UCH,0=auto;1=off;2=heat;2=water,,Boiler Modus
 hcmode2,UCH,0=off;3=heat;4=water,,Boiler Modus
 mcmode,UCH,0=disabled;1=on;2=off;3=auto;4=eco;5=low,,Mischer Modus
-mctype,UCH,0=inactive;1=mixer;2=fixed;3=hwc;4=returnincr;5=pool,,Mischer Typ
-mctype7,BI0:7,0=inactive;1=mixer;2=fixed;3=hwc;4=returnincr;5=pool,,Mischer Typ
+mctype,UCH,0=inaktiv;1=Heizen;2=Festwert;3=WW;4=Rückl.anh.;5=Pool,,Mischer Typ
```

Comment:
lookup value assignments in templates are intentionally kept identical in all languages in order to be able to switch the language without having to adjust several clients. you'd rather do a translation of these values in your own client/frontend.